site/docs/latest/cookbooks/compaction.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Topic compaction cookbook
+tags: [admin, clients, compaction]
+---
+
+Pulsar's [topic compaction](../../getting-started/ConceptsAndArchitecture#compaction) feature enables you to create **compacted** topics in which older, "obscured" entries are pruned from the topic, allowing for faster reads through the topic's history (which messages are deemed obscured/outdated/irrelevant will depend on your use case).
+
```

Comment:
A simple motivating usecase would be helpful here. The one I'm going to use for an upcoming is a stock ticker. Some users will want to see just the latest value for each stock, while some want to see the whole history of how the prices changed over time.